src/templates/Homepage/index.tsx

```diff
@@ -113,64 +143,113 @@ const Homepage: React.FC = () => {
               {text}
             </ic-typography>
           ))}
-          <ic-typography variant="body">
-            <ic-link href={withPrefix(homepage.about.link)}>
-              {homepage.about.linkText}.
-            </ic-link>
-          </ic-typography>
+          <ic-button href={withPrefix(homepage.about.link)}>
+            {homepage.about.buttonText}
+          </ic-button>
         </div>
         <hr aria-hidden="true" className="divider" />
-        <div className="section">
-          <ic-typography variant="h2" apply-vertical-margins="true">
-            <h2>{homepage.contribute.title}</h2>
-          </ic-typography>
-          {homepage.contribute.content.map((text: string, index: number) => (
-            <ic-typography
-              variant="body"
-              data-class="text-content"
-              apply-vertical-margins="true"
-              // eslint-disable-next-line react/no-array-index-key
-              key={index}
-            >
-              {text}
+        <div className={clsx("section", "display-row")}>
+          <div>
+            <ic-typography variant="h2" apply-vertical-margins="true">
+              {homepage.design.title}
             </ic-typography>
-          ))}
-          <ic-typography variant="body">
-            <ic-link href={withPrefix(homepage.contribute.link)}>
-              {homepage.contribute.linkText}.
-            </ic-link>
-          </ic-typography>
+            {homepage.design.content.map((text: string, index: number) => (
+              <ic-typography
+                variant="body"
+                data-class="text-content"
+                apply-vertical-margins="true"
+                // eslint-disable-next-line react/no-array-index-key
+                key={index}
+              >
+                {text}
+              </ic-typography>
+            ))}
+            <div className="homepage-button-container">
```

Comment:
I think these buttons (and the two in the developer section) should move to be above and below each other at smaller screen sizes because the second one gets cut off on the right (e.g. at 430px and below)